emgapiv2/api.py

```diff
@@ -237,6 +237,37 @@ def list_mgnify_analyses(request):
     return qs
 
 
+#################################################################
+#                                                               #
+#                          My Data                             #
+#                                                               #
+#################################################################
+
+
+@api.get(
+    "/my-data/analyses",
+    response=List[MGnifyAnalysis],
+    tags=[ApiSections.ANALYSES.value],
+    summary="List all private analyses (MGYAs) available from MGnify",
+    operation_id="list_private_mgnify_analyses",
+)
+def list_private_mgnify_analyses(request):
+    qs = analyses.models.Analysis.objects.private_only()
+    return qs
+
+
+@api.get(
+    "/my-data/studies",
+    response=List[MGnifyStudy],
+    tags=[ApiSections.STUDIES.value],
+    summary="List all private studies available from MGnify",
+    operation_id="list_private_mgnify_studies",
+)
+def list_private_mgnify_studies(request):
+    qs = analyses.models.Study.objects.private_only()
+    return qs
```

Comment:
We need an ownership check here, else it lets anybody list all private objects.
I think we want an extra manager method on the privacy manager base, like `def owned_only(webin): qs = qs.filter(webin_submitter__iexact=webin)`, and then here we can do `qs = analyses.models.Study.objects.owned_only(request.user)` (assuming we also add an auth layer)

Reply:
Thanks @SandyRogers 
Actually.. I left this as is on purpose. 
you're definitely right that we need an ownership check / some sort of middleware here. 
However, I just put this as sort of a placeholder. Currently I just used it to double check that private/public data are indeed being filtered.
And yes, my plan was also to add the auth layer in the future.